### ISSUE TITLE
Breaks the package Nova Inline Relationship

### ISSUE THREAD

**Issue by jberculo** [tanmuhittin/nova-scroll-top#10]
Hi,

Just a heads up that this package breaks https://novapackages.com/packages/kirschbaum-development/nova-inline-relationship.

When submitting a form with for example a HasOne, it fails with 'undefined key: values'.

Regards,
Joost.

**Comment by ragingdave** [comment]
Are you certain that it's specifically this package at fault? It seems the js in this repo has no reference to values, whereas the `resources/js/components/RelationshipFormItem.vue` code in the noted repo, does reference values. I would recommend removing this package temporarily and seeing if the problem persists.

**Comment by jberculo** [comment]
Yes I am, removing your package solved the issue. It of course could be something wrong in the nova inline relationships package, but that one is at present dead in the water.  Removing that package is no option.
